### PR TITLE
minor corrections of comments

### PR DESCRIPTION
**Pull request by anton-vinogradov-su**
## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (would cause existing functionality to change)

## Wiki Update

- [ ] This change requires an update to the Wiki.
- [ ] This change does not require an update to the Wiki.

## Checklist
- [ ] This change follows the code style of this project.
- [ ] This change only changes what is necessary to the feature.

### CODE REVIEW

File: PokeAlarm/Alarms/Twilio/TwilioAlarm.py [hunk 1]
```diff
@@ -33,15 +33,15 @@ class TwilioAlarm(Alarm):
                        "Lure will expire at <24h_time> (<time_left>)."
         },
         'gym': {
-            'message': "A Team <old_team> gym has fallen!"
-                       "It is now controlled by <new_team>. <gmaps>"
+            'message': "A Team <old_team> gym has fallen! <gmaps>"
```
Comment (kvangent) [inline PokeAlarm/Alarms/Twilio/TwilioAlarm.py]: Why did you move gmaps up a line?

Comment (anton-vinogradov-su) [reply PokeAlarm/Alarms/Twilio/TwilioAlarm.py]: I didn't. My changes in TwilioAlarm.pl only at line 83.

Comment (kvangent) [reply PokeAlarm/Alarms/Twilio/TwilioAlarm.py]: Sorry about that, it was a change that was done in an earlier commit that hadn't been made to dev yet.
